File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        // Brute force Method
        
        // Declare new array
        int [] arr = new int[2];
        // Iterate through nums array
        for(int i = 0; i < (nums.length - 1); i++){
            // Iterate through all following numbers
            for(int j = i + 1; j < nums.length; j++){
                // Test if numbers add up to target
                if(nums[i] + nums[j] == target){
                    arr[0] = i;
                    arr[1] = j;
                }
            }
        }
        return arr;
    }
}

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    // Create new instance of the Map prototype
    let mp = new Map()
    // Iterate through nums array
    for (let i = 0; i < nums.length; i++) {
        let diff = target - nums[i]
        
        // look and see if mp has the difference
        if (mp.has(diff)) {
            return [i, mp.get(diff)]
        }
        
        mp.set(nums[i], i)
    }
};